templates/display-cans-and-bottles.html

```diff
@@ -8,28 +8,33 @@
 
 <table>
 	<colgroup>
-		<col width="25%" />
-		<col width="25%" />
-		<col width="25%" />
-		<col width="25%" />
+		<col width="50%" />
+		<col width="16.66%" />
+		<col width="16.66%" />
+		<col width="16.66%" />
 	</colgroup>
 	<thead>
 		<tr>
-			<th>Manufacturer</th>
 			<th>Name</th>
 			<th>ABV</th>
 			<th>Price</th>
-			<!-- <th>Remaining</th> -->
+			<th>Stock status</th>
 		</tr>
 	</thead>
 	<tbody>
-		{% for t in types %}
+		{% for l in lines %}
```

Comment:
This was called "types" because the objects being passed are StockType class instances.

Reply:
(It's "lines" in the on-tap template because the objects are StockLine class instances.)